cmd/snapctl/commands.go

```diff
@@ -117,6 +117,21 @@ var (
 						flRunning,
 					},
 				},
+				{
+					Name: "config",
```

Comment:
a nice doc for the current state and the future state, that would be nice. It has nothing to do with your fix here. I'm just trying to understand it. thanks.